src/Transpiler.ts

```diff
@@ -305,6 +309,9 @@ export abstract class LuaTranspiler {
 
     public transpileLuaLibFunction(func: LuaLibFeature, ...params: string[]): string {
         this.importLuaLibFeature(func);
+        params = params.filter(element => {
```

Comment:
I still don't understand in what situation element.tostring() would be empty string, could you give an example?

Reply:
sure, I hit this when calling `array.unshift()` with no params